build-system/tasks/karma.conf.js

```diff
@@ -132,7 +132,7 @@ module.exports = {
   ],
 
   // Number of sauce platforms to start in parallel
-  concurrency: 4,
+  concurrency: Infinity,
```

Comment:
From the Karma [docs](http://karma-runner.github.io/3.0/config/configuration-file.html), `Infinity` is the default concurrency, so you can just remove this config setting.

Reply:
Removed.